src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -348,6 +355,10 @@ void UserInfoPopup::updateUserData()
 
         this->userId_ = id;
 
+        this->ui_.userIDLabel->setText(TEXT_USER_ID + this->userId_);

```

Comment:
```
if (this->ui_.userIDLabel)
{
    this->ui_.userIDLabel->setText(TEXT_USER_ID + this->userId_);
}
```

Reply:
I don't understand why do that. Can you please explain?

Reply:
Oh, sorry, I forgot to explain.

`this->ui_.userIDLabel` exists only when `getSettings()->showUserIDsInUsercards` is `true`. So when it is `false`, then `this->ui_.userIDLabel` is `nullptr` and you call `setText()` from empty object.

I just built your branch, launched the chatterino, opened the usercard, and got crash just because the `showUserIDsInUsercards` setting was not set.

Reply:
Oh thank you for expalaining. I will fix it ASAP